src/main/webapp/renderers/Step2Renderer.js

```diff
@@ -0,0 +1,59 @@
+import RendererInterface from './RendererInterface.js';
+
+export default class Step2Renderer extends RendererInterface {
+    constructor(parentElement) {
+        super(parentElement);
+    }
+
+    generateInfoPane(prevEvent, nextEvent, index) {
+        if(prevEvent.location == nextEvent.location) {
+            return `
+                <div class="info-pane">
+                    <p class="same-location-msg">Same location</p>
+                </div>
+            `;
+        } else {
```

Comment:
Can we remove this else as we are saying if "a" return something. So by default the rest is not "a"

Reply:
Done! Can you have a look?